### ISSUE TITLE
conda-pack for offline install

### ISSUE THREAD

**Issue by minrk** [yuvipanda/the-littlest-jupyterhub#33]
[conda-pack](https://github.com/jcrist/conda-pack) may provide what we need for a single-file downloadable installer instead of constructor, since it supports conda + pip, etc.

**Comment by yuvipanda** [comment]
+1, I like it! With this, the bootstrap script could just be 'download this tarball, extract, then run installer'. But I'd like to keep the current option too, since that makes development so much easier.

**Comment by minrk** [comment]
Makes sense. Do we want to start the install with a frozen environment instead of just the leaf packages?
